python/src/scipp/neutron/instrument_view.py

```diff
@@ -9,13 +9,19 @@ def instrument_view(scipp_obj=None, positions="position", **kwargs):
     detector vector positions.
     Use the `positions` argument to change the vectors used as pixel positions.
     Sliders are added to navigate extra dimensions.
-    Spatial slicing and pixel opacity control is available using the controls
-    below the scene.
-    Use the `pixel_size` argument to specify the size of the detectors.
-    If no `pixel_size` is given, a guess is performed based on the extents of
-    the positions.
-    The aspect ratio of the positions is preserved by default, but this can
-    be changed to automatic scaling using `aspect="equal"`.
```

Comment:
So i updated the doc string in my PR (https://github.com/scipp/scipp/pull/1537/files#diff-363ad19e0ca6011d877b557e14ece009d135089a0349acea5548fb447d50839a) but this reverts it to the previous state.